doc/guides/timers-in-node.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Timers in node.js
+layout: docs.hbs
+---
+
+# Timers in node.js and beyond
+
+Timers are a collection of global functions in node.js that allow you to 
+execute code after a set period of time. To fully understand when timer 
+functions will be executed, it's a good idea to read up on the the node 
+[Event Loop](https://nodesource.com/blog/understanding-the-nodejs-event-loop/).
+
+## Code Time Machine
```

Comment:
Going to rework this a bit to provide a better introduction to the `set` functions.